aiida_workgraph/engine/workgraph.py

```diff
@@ -782,18 +782,28 @@ def run_tasks(self, names: t.List[str], continue_workgraph: bool = True) -> None
                         )
                     )
                     continue
+            # This calcfuntion/workfunction is already run
```

Comment:
This seems a bit hacky. Why is the work graph in a situation where it is inspecting a calcfunction/workfunction that has already run?